language/bytecode-verifier/transactional-tests/tests/control_flow/break_nested.mvir

```diff
@@ -5,14 +5,19 @@ main() {
     let y: u64;
     x = 0;
     y = 0;
-    while (true) {
-        loop {
-            y = 5;
-            break;
-        }
-        x = 3;
-        break;
-    }
+
+@outer_loop:
+    goto @inner_loop;
+
+@inner_loop:
+    y = 5;
+    goto @outer_loop_b0;
+
+@outer_loop_b0:
+    x = 3;
+    goto @exit;
+
+@exit:
```

Comment:
This @ syntax feels a bit confusing since it corresponds to addresses in the source language, e.g. `let x: address = @0x42`

Thoughts about just `label <ident>`, we could probably also then kill the `@` on the goto side of things

Reply:
Sure, that works for me!